src/components/attendee/VideoComms/internal/scss/TwilioTrackDisplay.module.scss

```diff
@@ -0,0 +1,3 @@
+.mirrored {
+  transform: scale(-1, 1);
```

Comment:
[NOTE]
There should be a way to do the mirroring effect w/o flipping the DOM video element controls as well. Maybe through `filter` instead of `transform`, but I think there should be a flip on the video itself, before the stream is picked up by the DOM element.